docs/guide/rest-resources.md

```diff
@@ -147,22 +147,40 @@ Typically, you should return at least the `self` link representing the URL to th
 
 ```php
 use yii\db\ActiveRecord;
-use yii\web\Link;
+use yii\web\Link; // represents a link object as defined in JSON Hypermedia API Language.
 use yii\web\Linkable;
 use yii\helpers\Url;
 
-class User extends ActiveRecord implements Linkable
+class UserResource extends Model implements Linkable
```

Comment:
why not use active record? also if not using it, the import above should be changed.